data/persian/bbc_persian_radio/liveradio.json

```
@@ -10,11 +12,14 @@
     "lastPublished": 0,
     "options": {},
     "analyticsLabels": {
+      "pageTitle": "رادیو فارسی بی‌بی‌سی - BBC News فارسی",
       "pageIdentifier": "persian.bbc_persian_radio.liveradio.page",
-      "pageTitle": "رادیو فارسی بی‌بی‌سی - BBC News فارسی"
+      "producerId": "69",
+      "contentType": "player-live",
+      "producer": "PERSIAN "
```

```suggestion
      "producer": "PERSIAN"
```

Just to double check are persian and dari meant to be using the same producer values?